Makefile

```
@@ -634,12 +642,24 @@ endif
 
 deploy-webhook: manifests kustomize ## Using helm, deploy just the webhook in the k8s cluster
 ifeq ($(DEPLOY_WITH), helm)
-	helm install $(DEPLOY_WAIT) -n $(NAMESPACE) --create-namespace $(HELM_RELEASE_NAME) $(OPERATOR_CHART) --set image.repo=null --set image.name=${OPERATOR_IMG} --set image.pullPolicy=$(HELM_IMAGE_PULL_POLICY) --set imagePullSecrets[0].name=priv-reg-cred $(HELM_OVERRIDES) --set webhook.enable=true,controllers.enable=false
+	helm install $(DEPLOY_WAIT) -n $(NAMESPACE) --create-namespace $(HELM_RELEASE_NAME) $(OPERATOR_CHART) --set image.repo=null --set image.name=${OPERATOR_IMG} --set image.pullPolicy=$(HELM_IMAGE_PULL_POLICY) --set imagePullSecrets[0].name=priv-reg-cred $(HELM_OVERRIDES)
 	scripts/wait-for-webhook.sh -n $(NAMESPACE) -t 60
 else
 	$(error Unsupported deployment method for webhook only: $(DEPLOY_WITH))
 endif
 
+.PHONY: deploy-prometheus
+deploy-prometheus:
+	helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+	helm repo update
+	helm install $(DEPLOY_WAIT) -n $(PROMETHEUS_NAMESPACE) --create-namespace $(PROMETHEUS_HELM_NAME) $(PROMETHEUS_CHART) --values helm-charts/prometheus/values.yaml $(PROMETHEUS_HELM_OVERRIDES)
+	scripts/deploy-prometheus.sh -n $(PROMETHEUS_NAMESPACE) -l $(PROMETHEUS_HELM_NAME) -a deploy -u $(PROMETHEUS_USERNAME) -p '$(PROMETHEUS_PASSWORD)' -d $(PROMETHEUS_DBNAME)
```

```suggestion
	scripts/deploy-prometheus.sh -n $(VDB_NAMESPACE) -l $(PROMETHEUS_HELM_NAME) -a deploy -u $(PROMETHEUS_USERNAME) -p '$(PROMETHEUS_PASSWORD)' -d $(PROMETHEUS_DBNAME)
```

Also, this must be a separate command, there will be one prometheus instance and one service monitor per vdb.

---

addressed in https://github.com/vertica/vertica-kubernetes/pull/1013/commits/4cd17545f32cb85134bfdedc6a51e82a8b5549b1